app/src/Main.elm

```diff
@@ -220,7 +266,16 @@ footer =
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
-    Sub.batch
-        [ Sock.listen (webSocketUrl model.flags) Socket
-        , storageGot SetId
-        ]
+    case model.route of
+        Route.Menu ->
+            Sub.batch
+                [ Sock.listen (webSocketUrl model.flags) Socket
+                , storageGot SetId
+                , Sub.map MenuMsg (Menu.subscriptions model.menu)
```

Comment:
The base subscriptions need pulling out, then only have to append the difference